podcasts/Zendesk/MessageSupportView.swift

```diff
@@ -72,10 +72,15 @@ struct MessageSupportView: View {
                     viewModel.completion = nil
                     dismiss?()
                 }))
-            case .failure:
-                return Alert(title: Text(L10n.supportErrorTitle), message: Text(L10n.supportErrorMsg), dismissButton: .default(Text(L10n.supportOK), action: {
-                    viewModel.completion = nil
-                }))
+            case .failure(let error):
+                switch error {
+                case PlaybackError.errorDuringPlayback:
+                    return Alert(title: Text(L10n.supportWatchHelpTitle), message: Text(L10n.supportWatchHelpMessage), primaryButton: .default(Text(L10n.supportWatchHelpOpenedApp)) { viewModel.submitRequest() }, secondaryButton: .default(Text(L10n.supportWatchHelpSendWithoutLog)) { viewModel.submitRequest(ignoreUnavailableWatchLogs: true) })
```

Comment:
I wonder, should we use `viewModel.submitRequest(ignoreUnavailableWatchLogs: true)` for both actions to handle for a case where the user has opened the watch app but we still weren't able to get logs for it. 

I think currently from the users perspective they may feel like they're stuck in a loop and may keep tapping the "I opened the watch app" button but continually end up being prompted.